src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -3959,3 +3992,21 @@ export function showQueryUpdatedToast(message: string) {
         theme: 'light',
     } as any);
 }
+
+export function getDataTypeByDataFilterValues(
+    dataFilterValues: DataFilterValue[]
+): DataType {
+    let valueCount = dataFilterValues.length;
+
+    _.map(dataFilterValues, (dataFilterValue: DataFilterValue) => {
```

Comment:
this isn't a mapping operation so it shouldn't use map.  only use map with the iterator returns a value.   this is really a each loop.  And in fact, I believe it's really a case for ```.some()``` because i think the function says, if at least one member of the array is undefined or empty string, then the type is Number.  In that case, we should exit the loop as soon as we detect one of these.  That's what some will do.